### ISSUE TITLE
options are not sticky after log out

### ISSUE THREAD

**Issue by monzug** [alpheios-project/components#796]
as user test, I changed options in UI panel and Features. they worked fine during the session. when I logged out, then logged in again as user test, default option were applied to all options.

**Comment by balmas** [comment]
interesting. safari is throwing an error storing settings:

[Error] Unexpected error storing Alpheios option alpheios-feature-options__2__lookupLanguage: TypeError: t.includes is not a function. (In 't.includes(r)', 't.includes' is undefined)
	(anonymous function) (content-safari.js:1:2066990)
	promiseReactionJob

Investigating.

**Comment by monzug** [comment]
I think on log out it goes to default

**Comment by balmas** [comment]
oddly, I can't reproduce either of these errors in qa-3.0.0.66/components 1.2.41 and I see that my settings are saved and reloaded after login/logout/login

**Comment by balmas** [comment]
oh... but I can if I use a production build of the components library, rather than a development build.

@kirlat I think Safari is having some problems with the minified code.

**Comment by kirlat** [comment]
This issue seems to be very sneaky: sometimes I see it, sometimes I don't. I was not able to say what is exactly the cause of it. I did some minor build optimizations (switched to using `vue` profile instead of `vue-postcss` because now we transpile CSS in components and just copy the resulting file to webextensions; I've also removed related dependencies). I'm not sure if it made any difference or not. Now I, however, see the correct build being produced. The error has gone for me completely. 

I think we can probably stick to the current configuration and keep an eye on it. If we'll see the same error message in the logs, we'll know what to check, but I hope we won't. What do you think?

**Comment by balmas** [comment]
ok, the only other idea I had was to try changing the name of one of the variables, thinking that maybe whatever algorithm is being used was having trouble with the fact that both started with 'keys'

**Comment by kirlat** [comment]
That can be an idea worth trying. I will try that if the problem will occur again.

**Comment by monzug** [comment]
I just retested the Options in Chrome and FF and they are not sticky there too. it used to work. 
I login as user test, change Otions in UI, check they work fine in current session, log out and log in again as user test. the Options are back to default.

**Comment by monzug** [comment]
in Chrome I did install Alpheios Reading Tools 3.0.0.60
Alpheios Components 1.2.33 and Options are sticky on login/logout. something has changed between 3.0.0.60 and 3.0.0.66

**Comment by monzug** [comment]
test this morning by installing few builds. it looks like in both 3.0.0.63 and 3.0.0.64 the options are sticky, so something in build 3.0.0.65 is the culprit.
interesting is that the user's option are override for the session in build 3.0.0.65 and 3.0.0.66 but they still work fine in the previous builds. so it looks like something is forcing the default options rather than override the saved options.

**Comment by balmas** [comment]
3.0.0.64 uses components commit 19526bcc7c000f5a3d5a86cd2c0255fbf46fc28d
3.0.0.65 uses components commit 09f317fc7630420d2e54ac7b957273ee90b36e77

I don't see anything changes in components that would cause this. Will look at the webextension changes next.

**Comment by balmas** [comment]
We believe we have traced this to a problem in the package we use for minifying code for production. https://github.com/terser/terser/issues/452 , probably exposed by a change to the webextension code in the first merge of the safari changes, where we switched to using `const` to define a number of variables, in keeping with updates we have made elsewhere in the code. An update to the latest release of terser, 4.3.1 appears to fix the problem.
